src/averageAges.js

```diff
@@ -14,14 +13,22 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
-}
+  const men = people.filter((person) => {
+    return century
+      ? person.sex === 'm' && century === Math.ceil(person.died / 100)
```

Comment:
```suggestion
const MAN = 'm';
const CENTURY = 100;

const isMan = (person) => person.sex === MAN;
const getCentury = (person) => Math.ceil(person.died / CENTURY);

      ? isMan(person) && century === getCentury(person);
```

* Avoid using magic strings and numbers - it's a programming antipattern. Consider using constants with meaningful names instead. 
* Also, consider using helper functions to move repeating logic there. It's DRY - don't repeat yourself principle.